.ci/init.gradle

```diff
@@ -36,6 +46,14 @@ final Vault vault = new Vault(
         .build()
 )
         .withRetries(5, 1000)
+    
+Process process = Native.get(Process.class)
```

Comment:
Ug, I'm not sure about mutating the environment here. This might cause weird behavior with concurrently executing nested BWC builds.

Reply:
Also, I don't think this practically actually provides any additional security for the reasons mentioned here: https://github.com/elastic/infra/pull/13780#issuecomment-521380050

Reply:
We are only really mutating if we are transforming the IDs into a token, should restructure the code to better reflect that.

Reply:
That will only happen on the first invocation of the script

Reply:
Won't the bwc tests run in their own demons ? As a demon will only run one build right ?